src/main/java/org/hobbit/utils/rdf/RdfHelper.java

```diff
@@ -242,6 +242,206 @@ public static Duration getDurationValue(Model model, Resource subject, Property
+     * @param predicate
+     *              the predicate of the triple. <code>null</code> works like a wildcard.
+     * @return object of the triple as a Boolean or false
+     *         if such a triple couldn't be found.
+     */
+    public static boolean getBooleanValue(Model model, Resource subject, Property predicate) {
+        if (model == null) {
+            return false;
+        }
+        Literal literal = getLiteral(model, subject, predicate);
+        if (literal != null) {
+            try {
+                return literal.getBoolean();
+            } catch (Exception e) {
+                // nothing to do
+                LOGGER.debug("Exception occured. Returning false.", e);
+            }
+        }
+        return false;
+
+    }
+
+    /**
+     * Returns the objects as Float of all triples that have the given subject and
+     * predicate and that can be found in the given model.
+     *
+     * @param model
+     *              the model that should contain the triple
+     * @param subject
+     *              the subject of the triple. <code>null</code> works like a wildcard.
+     * @param predicate
+     *              the predicate of the triple. <code>null</code> works like a wildcard.
+     * @return object of the triple as a Float or 0F
+     *         if such a triple couldn't be found.
+     */
+    public static Float getFloatValue(Model model, Resource subject, Property predicate) {
+        if (model == null) {
+            return 0F;
+        }
+        Literal literal = getLiteral(model, subject, predicate);
+        if (literal != null) {
+            try {
+                return literal.getFloat();
+            } catch (Exception e) {
+                // nothing to do
+                LOGGER.debug("Exception occured. Returning 0F.", e);
+            }
+        }
+        return 0F;
+
+    }
+
+
+    /**
+     * Returns the objects as Double of all triples that have the given subject and
+     * predicate and that can be found in the given model.
+     *
+     * @param model
+     *              the model that should contain the triple
+     * @param subject
+     *              the subject of the triple. <code>null</code> works like a wildcard.
+     * @param predicate
+     *              the predicate of the triple. <code>null</code> works like a wildcard.
+     * @return object of the triple as a Double or 0D
+     *         if such a triple couldn't be found.
+     */
+    public static Double getDoubleValue(Model model, Resource subject, Property predicate) {
+        if (model == null) {
+            return 0D;
+        }
+        Literal literal = getLiteral(model, subject, predicate);
+        if (literal != null) {
+            try {
+                return literal.getDouble();
+            } catch (Exception e) {
+                // nothing to do
+                LOGGER.debug("Exception occured. Returning 0D.", e);
+            }
+        }
+        return 0D;
```

Comment:
`0` as a "sentinel value" is not a good idea since the user has the issue that he can not distinguish between a `0` that has been read from the data and a `0` representing the absence of the data.